src/hotkey.ts

```diff
@@ -47,9 +48,19 @@ export function eventToHotkeyString(
   }
 
   if (!modifierKeyNames.includes(key)) {
-    const nonOptionPlaneKey =
+    // MacOS outputs cymbols when `Alt` is held, so we map them back to the key symbol if we can
+    const altNormalizedKey =
       hotkeyString.includes('Alt') && matchApplePlatform.test(platform) ? macosSymbolLayerKeys[key] ?? key : key
-    const syntheticKey = syntheticKeyNames[nonOptionPlaneKey] ?? nonOptionPlaneKey
+
+    // MacOS outputs lowercase characters when `Command+Shift` is held, so we map them back to uppercase if we can
+    const shiftNormalizedKey =
+      hotkeyString.includes('Shift') && matchApplePlatform.test(platform)
```

Comment:
a thought: maybe we should only normalize when `Mod` part of the string: that way using the raw `Meta+Shift+lowercase` would still be an option?

Reply:
Although `Meta+Shift+lowercase` wouldn't work on Linux :(

Reply:
That's not a bad idea - I like the opt-in normalization you get there. `Meta` won't actually work on Linux at all I think.

Reply:
Ooh, but it doesn't work in this case because we are just converting events to hotkey strings - we don't yet know what the target string is that we will be comparing it to since this is a lower-level util.